packages/config/src/configs/moonbeam.ts

```diff
@@ -81,7 +81,7 @@ export const moonbeamConfig = new ChainConfig({
       contract: ContractBuilder().Xtokens().transfer(),
       destination: hydraDX,
       destinationFee: {
-        amount: 0.01,
+        amount: 0.05,
```

Comment:
Technically the fee is lower and the existential deposit (what we consider `minBalance`) is the one that makes it go high to .05 
![image](https://github.com/PureStake/xcm-sdk/assets/93129175/41b1b6a6-62cb-4d62-aff3-e505c3641989)
We would have to create another function in the `AssetMinBuilder` file
However seeing that it is not that significant and our current work load we could leave it like this